CHANGELOG.rst

```diff
@@ -35,18 +45,15 @@ Fixed
   ``service`` or ``action`` parameter. (bug fix) #4675
 
   Reported by James Robinson (Netskope and Veracode).
-* Replace ``sseclient`` library on which st2client and CLI depends on with ``sseclient-py``.
-  ``sseclient`` has various issue which cause client to sometimes hang and keep the connection open
-  which also causes ``st2 execution tail`` command to hang for a long time. (improvement) #4686
+* Replace ``sseclient`` library on which CLI depends on with ``sseclient-py``. ``sseclient`` has
+  various issue which cause client to sometimes hang and keep the connection open which also causes
```

Comment:
This sentence has some grammatical flaws.

Reply:
Nevertheless, I'm just moving the entries from one location to another to close the patch release. I'm not trying to fix grammar here.